index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dan's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section id="skeleton">
+      <header>
+        <h1 id="welcome">Welcome</h1>
+        <a href="#" id="menu">
+          <img src="images/menu.svg" alt="Drop down menu" />
+        </a>
+      </header>
+      <div id="page">
+        <div id="intro">
+          <p id="denis">Hey there. I’m Denis</p>
+          <p id="trade">I’m a software developer</p>
+        </div>
+        <div id="advert">
+          <p>
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! <br />
+            If you like what you see and have a project you need coded, <br />
+            don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="social">
```

Comment:
- [ ] Please wrap the headline section with section tag instead of div tag 
because semantically using section is better  If you are only using the element as a styling wrapper, use a div. but here the section is not just a styling wrapper it represents a sectioning part